### PR TITLE
Add gqty, & use it to fetch the organization's name

### PR DESCRIPTION
**Pull request by levity**
based on #353

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -43,7 +51,7 @@
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-empty-function": "off",
-    "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
```
Comment (exrhizo) [inline .eslintrc:54]: whats this do?

File: src/components/OrganizationHeader/OrganizationHeader.tsx [hunk 2]
```diff
@@ -92,10 +92,15 @@ const useStyles = makeStyles(theme => ({
...
+  name?: string;
   buttonText: string;
   onButtonClick: () => void;
 }
-export const OrganizationHeader = ({ buttonText, onButtonClick }: Props) => {
+export const OrganizationHeader = ({
+  name,
+  buttonText,
+  onButtonClick,
+}: Props) => {
```
Comment (exrhizo) [inline src/components/OrganizationHeader/OrganizationHeader.tsx:103]: in this case, `useSelectedCircle` ought to be passed in from parent as well (perhaps for another PR)

File: src/services/api.ts [hunk 3]
```diff
@@ -43,11 +45,26 @@ export class APIService {
...
-        }
-      : {};
+
+    const auth: Record<string, unknown> = {};
+    if (token) {
+      const authHeader = 'Bearer ' + token;
+      auth.headers = { Authorization: authHeader };
+
+      // add interceptor so GQty requests have proper authorization
+      this.interceptor = hasuraAxios.interceptors.request.use(
```
Comment (exrhizo) [inline src/services/api.ts:55]: Another thing that interceptors can be used for is triggering re-authorization when a token expires

File: .eslintrc (hunk 1 above)
Comment (levity) [reply .eslintrc:54]: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-ts-comment.md

the original doesn't seem to exist (maybe it got renamed)

**Review by exrhizo — APPROVED** [review]

**Pull request merged**